benchmarks/test/example.json

```diff
@@ -100,7 +99,9 @@
       0.05571739934384823,
       0.058703068643808365
     ],
-    "single_value": [1]
+    "single_value": [
+      1
+    ]
   },
   "outputs_file": null
-}
+}
```

Comment:
nit: is this removing the trailing newline?

Reply:
Done.